docs/csharp/roslyn-sdk/snippets/source-generators/SourceGenerator/SourceGenerator.csproj

```
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>11.0</LangVersion>
```

To avoid the new warning, let's enforce extended analyzer rules.

```suggestion
    <LangVersion>11.0</LangVersion>
    <EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>
```